content/en/docs/ops/deployment/multicluster-considerations/index.md

```diff
@@ -0,0 +1,155 @@
+services configure this server.
+
+Istio ignores the IP address returned from the DNS lookup, however. Instead, it
+uses the list of active endpoints for the host and load balances across them. Istio uses either Kubernetes
+`Service`/`Endpoint` or Istio `ServiceEntry` to configure its internal mapping of hostname to IP addresses.
+
+This two-tiered naming system becomes more complicated when you have multiple clusters. Istio is inherently
+multicluster-aware, but Kubernetes is not (today). Because of this, for a client in one cluster to make a request to
+a service in another cluster, there must be a Kubernetes service for the host defined in the client cluster in order
+for the DNS lookup to succeed. This is true even if there are no instances of that service's Pods running in the
+client cluster.
+
+To generalize this, you must deploy a Kubernetes service to each cluster that consumes that service. This ensures that
+regardless of where the request originates, it will pass DNS lookup and be handed to Istio for proper routing.
+
+{{< tip >}}
+There are a few efforts in progress that will help simplify the DNS story in the future:
+
+* Istio will soon support DNS interception for workloads with a sidecar proxy. This will allow Istio to perform
+DNS lookup on behalf of the application.
+* [Admiral](https://github.com/istio-ecosystem/admiral) is an Istio community project that provides a number of
+multicluster capabilities, including automatic creation of service DNS entries.
+* [Kubernetes Multi-Cluster Services](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api)
+is a Kubernetes Enhancement Proposal (KEP) that defines an API for exporting services to multiple clusters.
+{{< /tip >}}
+## Networks
+
+Istio uses a simplified definition of [network](/docs/ops/deployment/deployment-models#network-models) to refer to
+pods that have direct reachability. For example, by default all the pods in a single cluster are on a single network.
+
+In multicluster configurations it is not uncommon to have clusters residing in different networks, meaning pods
+in one cluster are not directly reachable from pods in another.
+
+In a multi-network scenario, each cluster will have to expose any services needed by workloads in other networks.
+Cloud vendors may provide options that will not require exposing services on the public internet. Such an option,
+if it exists and meets your requirements, will likely be the best choice.
+
+{{< tip >}}
+Istio currently only supports cross-network communication to workloads with an Istio proxy. This is due to the
+fact that we expose services at Ingress Gateway with TLS pass-through, which enables mTLS directly to the workload.
+A workload without an Istio proxy, however, will likely not be able to participate in mutual authentication with
+other workloads. For this reason, Istio filters out-of-network endpoints for proxyless services.
```

Comment:
This is good content!  wonder if we position as this is by design to ensure the traffic is secured among clusters.   When service A call service B across clusters, service A and B must both have istio proxies to ensure the mtls communication among the two.

Reply:
Good point. Reworded the first sentence a bit ... PTAL